passionscotch/app/models/user.rb

```diff
@@ -0,0 +1,42 @@
+require 'bcrypt'
+
+class User < ActiveRecord::Base
+  # Remember to create a migration!
+  has_many :reviews
+  has_many :wishlist
+
+
+
+  include BCrypt
+
+  validates :email, presence: true
+
+  def password
+    @password ||= Password.new(password_hash)
+  end
+
+  def password=(new_password)
+    @password = Password.create(new_password)
+    self.password_hash = @password
+  end
+
+  def create
```

Comment:
you should not have a create User instance method. A huge code smell is using http request params in your models.